src/components/shortuct-button/ShortcutButton.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { memo } from 'react';
+
+interface Props {
+  buttonText: string;
+  onButtonClick: () => void;
+}
+
+const shortcut = {
+  root: {
+    padding: '0 0.5rem',
+    minHeight: '28px',
```

Comment:
@zutshiy min-height generally should be the last resort to fix any height issues. Let's play around the padding only or line-height.

Also, let's use absolute px units for padding as `rem` might result in inconsistent spacing in the calendar component as the value would rely on the user's root font-size value. If someone has a bigger root font size they would see more spacing than someone who has a smaller root font size.